rusqlite_migration/src/asynch.rs

```diff
@@ -30,7 +30,7 @@ impl AsyncMigrations {
     #[must_use]
     pub fn new(ms: Vec<M<'static>>) -> Self {
         Self {
-            migrations: Migrations::new(ms),
+            migrations: Arc::new(Migrations::new(ms)),
```

Comment:
When an unrelated change is more than a one line change (here, it requires adding a bunch of `Arc::new` and `Arc::clone` in mulitple places), it is much easier for reviewers if that change is broken out in a separate PR. It's fine for this this time, but please keep this in mind for the future.